test/jdk/java/text/Format/DecimalFormat/CloneTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8354522
+ * @summary Check for cloning interference
+ * @run junit CloneTest
+ */
+
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.text.DecimalFormat;
+import java.util.concurrent.Executors;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class CloneTest {
+    // Specifically tests that when DecimalFormat is cloned after use with
+    // a long double/BigDecimal, clones will be independent. This is not an
+    // exhaustive test.
+    @Test
+    public void testCloneIndependence() {
+        AtomicInteger mismatchCount = new AtomicInteger(0);
+        DecimalFormat df = new DecimalFormat("#");
+        String _ = df.format(Math.PI); // initial use of formatter
```

Comment:
We should probably comment the importance of this line, as without it the test will pass without the fix. (It sets the _data_ array to a non null value).

Reply:
Done